File: tests/fixtures/ruby/excess_args.rb
def create_user(name, email, age, role, active, score, level, department)
  {
    name: name, email: email, age: age, role: role,
    active: active, score: score, level: level, department: department
  }
end

def simple_func(a, b)
  a + b
end

class UserService
  def initialize(name, email, db, cache, logger, config)
    @name = name
    @email = email
    @db = db
    @cache = cache
    @logger = logger
    @config = config
  end
end

File: tests/fixtures/lua/excess_args.lua
function createUser(name, email, age, role, active, score, level, department)
    local user = {}
    user.name = name
    user.email = email
    user.age = age
    user.role = role
    user.active = active
    user.score = score
    user.level = level
    user.department = department
    return user
end

function simpleFunc(a, b)
    return a + b
end

local UserService = {}
function UserService:init(name, email, db, cache, logger, config)
    self.name = name
    self.email = email
    self.db = db
    self.cache = cache
    self.logger = logger
    self.config = config
end
